Automattic/wp-calypso
# Plans: update illustrations in My Plan feature cards.

Update to the illustrations of the feature cards (replacing gridicons, repeated icons and use updated illustrations).

## To test:

- checkout this branch on your local Calypso or use calypso.live,
- verify that the illustrations are correct at `http://calypso.localhost:3000/plans/:site` for both .com and JP plans.

 ref: p6TEKc-1SF-p2
